src/drt/include/triton_route/TritonRoute.h

```diff
@@ -75,6 +75,7 @@ struct ParamStruct
 {
   std::string outputMazeFile;
   std::string outputDrcFile;
+  int drcReportIterStep;
```

Comment:
More modern C++ idiomatic to use use std::option instead of magic numbers?

I find std::option to be more self-documenting than magic numbers...

Reply:
You can't swig a std::optional but I'll change it in the c++